src/App.jsx

```diff
@@ -55,15 +57,25 @@ function App() {
     setTodoList(newTodoList);
   }
   return (
-    <>
-      <h1>Todo List</h1>
-      <TodoForm onAddTodo={addTodo} />
-      {isLoading ? (
-        <p>Loading...</p>
-      ) : (
-        <TodoList onRemoveTodo={removeTodo} todoList={todoList} />
-      )}
-    </>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/"
+          element={
+            <>
+            <h1>Todo List</h1>
+        <TodoForm onAddTodo={addTodo} />
+        {isLoading ? (
+          <p>Loading...</p>
+        ) : (
+          <TodoList onRemoveTodo={removeTodo} todoList={todoList} />
+        )}   
+            </>
```

Comment:
I know this may not have been in the instructions, but it's good practice to pull components like this out.  Either to another file, or to a sub component like this:

~~~
const home = (
<>
            <h1>Todo List</h1>
        <TodoForm onAddTodo={addTodo} />
        {isLoading ? (
          <p>Loading...</p>
        ) : (
          <TodoList onRemoveTodo={removeTodo} todoList={todoList} />
        )}   
            </>
)

(later)
<Route path="/" element={home} />
~~~

It's much easier to read that way.